src/edu/csus/ecs/pc2/core/Utilities.java

```diff
@@ -159,11 +161,25 @@ public static String getSecretDataPath(String baseCDPPath, String problemShortNa
     }
 
     /**
-     * Return CCS path for input data and answer file names.
+     * Return CLICS path for sample data and answer file names.
      */
     public static String getSecretDataPath(String baseCDPPath, Problem problem) {
         return getSecretDataPath(baseCDPPath, problem.getShortName());
     }
+    
+    /**
+     * Return CLICS path for sample data and answer file names.
+     */
+    public static String getSampleDataPath(String baseCDPPath, String problemShortName) {
+        return baseCDPPath + File.separator + problemShortName + File.separator + SAMPLE_DATA_DIR;
+    }
+
+    /**
+     * Return CLICS path for input data and answer file names.
```

Comment:
This comment should say "... for **_sample_** data and answer file names, not "input" names...